Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/model/ContactDiaryLocation.kt

```diff
@@ -0,0 +1,9 @@
+package de.rki.coronawarnapp.contactdiary.model
+
+interface ContactDiaryLocation {
```

Comment:
Do we really need `ContactDiary` as a prefix for all classes? Isn't this implied by the namespace?